IPython/sphinxext/ipython_directive.py

```diff
@@ -138,7 +138,13 @@
 
 # Third-party
 from docutils.parsers.rst import directives
-from sphinx.util.compat import Directive
+import sphinx
+# sphinx.version_info available since 1.2, and we require 1.3
+# sphinx.util.compat deprecated at 1.6 and removed at 1.7
+if sphinx.version_info < (1, 5, 7):
+    from sphinx.util.compat import Directive
+else:
+    from docutils.parsers.rst import Directive
```

Comment:
I checked it was there in Docutils 0.10.1 before pushing the PR, I will check now exact version (probably from dawn of time ;-) )